### PR TITLE
Point and line rendering support

### PR DESCRIPTION
**Pull request by bilke**
Some additions to enable point and line rendering. Successfully tested with 
[primitives.gltf.zip](https://github.com/Siccity/GLTFUtility/files/4181324/primitives.gltf.zip).

Initial implementation by @ozgurozansen.

### CODE REVIEW

File: Scripts/Spec/GLTFMesh.cs [hunk 1]
```diff
@@ -6,6 +6,7 @@
 using Newtonsoft.Json;
 using UnityEngine;
 using UnityEngine.Scripting;
+using UnityEngine.Windows.WebCam;
```
Comment (Siccity) [inline Scripts/Spec/GLTFMesh.cs]: What's this doing in here?

Comment (bilke) [reply Scripts/Spec/GLTFMesh.cs]: Removed.

File: Scripts/Spec/GLTFMesh.cs [hunk 2]
```diff
@@ -267,6 +295,10 @@ private class BlendShape {
 						continue;
 					}
 
+					RenderingMode mode = RenderingMode.TRIANGLES;
+					for (int k = 0; k < meshes[i].primitives.Count; k++)
+						mode = meshes[i].primitives[k].mode;
+
```
Comment (Siccity) [inline Scripts/Spec/GLTFMesh.cs]: I understand that while GLTF supports different rendering modes per primitive, Unity only supports it per-mesh. This means you have to pick just one primitive mode and use that on the entire mesh. But why are you looping through all primitives to get it? Why not just get the first with
`mode = meshes[i].primitives[0].mode`?

Additionally, it doesn't seem like you use this mode value anywhere. Remove it?

Comment (bilke) [reply Scripts/Spec/GLTFMesh.cs]: Yep, this was a left-over from a previous implementation. I will remove it. Unity supports the mode at the submesh-level.

**Pull request merged**